src/base/dof_map_constraints.C

```diff
@@ -15,10 +15,8 @@
 // License along with this library; if not, write to the Free Software
 // Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 
-// Local includes
-#include "libmesh/dof_map.h"
-
 // libMesh includes
+#include "libmesh/dof_map.h"
```

Comment:
Let's avoid changes like this in the future?  I like our idiom of "the header with our definitions comes before other libMesh headers", since that makes it harder to accidentally write dependencies into a header file which get satisfied by source files' order-of-inclusion rather than by the header's own includes ... but if we just put such headers at the top of the libMesh section rather than in their own section, I'm sure I'll eventually end up breaking that idiom in subsequent PRs by alphabetizing without thinking about it.

Reply:
OK, my bad. I was thinking that we had possibly been inconsistently using "// local includes" and "// libmesh includes" in various places and was trying to make it consistent, but I see the idea now. Fixed in 071e2a4119ef01f65240a5d82c1501d215e2dce5